### ISSUE TITLE
[Cataclysm Trinket - Heart of Unliving]

### ISSUE THREAD

**Issue by Wexez** [Arkania/ArkCORE-NG#95]
Heart of Unliving - http://cata.openwow.com/item=77976 only procs from direct heals (Flash heal etc)
- It doesn't proc from HoTS (renew, etc) like it should.

For more info read http://bug.omfg.gg/issues/16764

**Comment by graham79** [comment]
I have fixed procs on trinkets before by adding an entry in spell_proc_event and adding the correct procFlags. However, the spell used by the trinket does give proc flags of 0x00004000. and tip say 14 successful cast positive magic spell. (positive=heals).

**Comment by Wexez** [comment]
Yeah I also tried to mess with spell_proc_event but it didn't work - only on direct heals. @graham79

**Comment by graham79** [comment]
I looked around to check on the proc data and haven't found anywhere where it says the spell procs from HoT's. Looking in the DBC entry it veryclearly says procs on 14 successfull spell Casts. and even gives the proc flags 0x00004000, which is the proc on successfull heal cast.
but if you want to use both dot and cast flags the two added together gives you a value =278528;
Try that in spell_proc_event and test it?

**Comment by Lbniese** [comment]
Any update? Did the 278528 spell_proc_event value work?
